### ISSUE TITLE
Unexpected strict mode reserved word?

### ISSUE THREAD

**Issue by stevenmcastano** [nfarina/homebridge#859]
```
/opt/apps/homebridge/node_modules/hap-nodejs/lib/util/eventedhttp.js:62
    let port = this._tcpServer.address().port;
    ^^^
SyntaxError: Unexpected strict mode reserved word
    at exports.runInThisContext (vm.js:73:16)
    at Module._compile (module.js:443:25)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/opt/apps/homebridge/node_modules/hap-nodejs/lib/HAPServer.js:11:25)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
```

Trying a fresh install have my old one of v0.1.1 stopped working.

**Comment by KhaosT** [comment]
please try update nodejs to a newer version. like v5.0 or later

Best Wishes,
Tian

**Comment by stevenmcastano** [comment]
Upgraded and now everything is running, new issue though with multi cast I think, I’ll open a new ticket.

*SMC

**Comment by ViViDboarder** [comment]
I'm getting this same issue using node v7.0.

It's a brand new install that I'm doing inside a container, so there should be little to no other influence. I've looked at the other closed issues that are related to this and they all say to use a newer version of node, so I've bumped mine up to 7.0 but it keeps happening.

Here is my [Dockerfile](https://github.com/ViViDboarder/docker-rpi-homebridge/blob/master/Dockerfile) that shows my install command, and [here](https://github.com/hypriot/rpi-node/blob/master/7.0/Dockerfile#L67) is where the base image is installing node. Is there something else that could cause this error beyond just the version of node? I have _very_ little experience with node myself, so it may be a silly question.
